content/influxdb/clustered/admin/bypass-identity-provider.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Bypass your identity provider
+description: >
+  InfluxDB clustered generates a valid management token (known as the _admin token_)
+  that can be used in development and testing environments in lieu of configuring
+  and using an OAuth2 identity provider.
+menu:
+  influxdb_clustered:
+    parent: Administer InfluxDB Clustered
+weight: 207
+---
+
+{{< product-name >}} generates a valid [management token](/influxdb/clustered/admin/tokens/#management-tokens)
+(known as the _admin token_) for managing databases and database tokens and
+stores it as a secret in your InfluxDB namespace.
+You can use the admin token with the [`influxctl` CLI](/influxdb/clustered/reference/cli/influxctl/)
+in lieu of configuring and using an OAuth2 identity provider.
+
+{{% warn %}}
+#### Do not use in production
+
+This feature is for development and testing purposes only and should not be used
+in a production InfluxDB cluster.
+{{% /warn %}}
+
+## Configure influxctl to use the admin token
+
+{{% code-placeholders "INFLUXDB_NAMESPACE|DIRECTORY_PATH" %}}
+
+1.  If you haven't already, [download, install, or upgrade to `influxctl` v2.2.0 or newer](/influxdb/clustered/reference/cli/influxctl/#download-and-install-influxctl).
+2.  Use `kubectl` to retrieve the admin token from your cluster namespace's
+    secret store and copy it to a file:
+
+    ```sh
+    kubectl get secrets/admin-token \
+      --template={{.data.token}} \
+      --namespace INFLUXDB_NAMESPACE | base64 -d > token.json
+    ```
+
+3.  Update your `influxctl` connection profile with a new `[profile.auth.token]`
+    section.
+4.  In the `[profile.auth.token]` section, assign the `token_file` setting to the location of your saved admin token file:
+
+    ```toml
+    [[profile]]
+    # ...
+      [profile.auth.token]
+        token_file = "/DIRECTORY_PATH/token.json"
+    ```
+{{% /code-placeholders %}}
+
+In the examples above, replace the following:
```

Comment:
This is totally fine if it's for consistency, and ignore my non-docs-expert suggestion as you see fit, but I do wonder if we should move these instructions above the numbered steps. Ideally people aren't just copying code from a doc and pasting it without looking it over, but even if they do look it over it may be confusing to not know what the values are ahead of time.

Reply:
We base most of our doc style and structure off the Google Technical Writing style guide which specifies placeholder descriptions should come after examples https://developers.google.com/style/placeholders#explain-placeholders.

This way, the first thing a reader sees is the code example itself, rather than a block of text explaining what to replace in code they haven't seen yet.

Reply:
Nice, thanks for sharing that! That's great to know what we use as a basis 💯